src/components/menu/js/menuServiceProvider.js

```diff
@@ -59,7 +59,7 @@ function MenuProvider($$interimElementProvider) {
       if (options.hasBackdrop) {
         options.backdrop = $mdUtil.createBackdrop(scope, "md-menu-backdrop md-click-catcher");
 
-        $animate.enter(options.backdrop, $document[0].body);
+        $animate.enter(options.backdrop, options.parent || $document[0].body);
```

Comment:
Backdrop is supposed to catch all clicks and the parent is usually smaller than the whole viewport; I suspect issues if `overflow: hidden` is used, etc. For cases where this is needed, how about a new option `backdropParent`? Then we wouldn't need a breaking change either.